moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -420,103 +365,130 @@ const ProjectView = () => {
                   <CircularProgress />
                 ) : (
                   <div className={classes.root}>
-                    <Box p={4} pb={2}>
+                    <Box
+                      sx={{
+                        pt: 2,
+                        px: 3,
+                      }}
+                    >
+                      <Breadcrumbs
+                        aria-label="breadcrumb"
+                        sx={{ color: "primary.main" }}
+                        separator={<NavigateNextIcon fontSize="small" />}
+                      >
+                        <Link color="inherit" href={allProjectsLink}>
+                          {`${
+                            previousProjectListViewQueryString
+                              ? "Filtered"
+                              : "All"
+                          } projects`}
+                        </Link>
+                        <Typography sx={{ color: "text.primary" }}>
+                          {`Project #${data.moped_project[0].project_id}`}
```

Comment:
Do y'all think "Project" is implied in the breadcrumb?

All projects > Project #3333
**or**
All projects > #3333